Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "NumShotOneTime", "type": "uint8" },
+        { "name": "NumAttackOneTime", "type": "uint8" }
+      ]
+    },
+    "EnemySpannerSpawner": {
+      "parent": "EnemySpinnerBaseSpawner",
+      "fields": [
+        { "name": "StartTime", "type": "float32" },
+        { "name": "AttackTime", "type": "float32" },
+        { "name": "AttackInterval", "type": "float32" }
+      ]
+    },
+    "EnemySparvaSpawner": {
+      "parent": "EnemyLarvaSpawner",
+      "fields": [
+      ]
+    },
+    "EnemySpinnerBaseSpawner": {
+      "parent": "EnemyBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "EnemySpinnerBaseSpawner::Type" },
+        { "name": "AlivePoint", "type": "vector3" },
+        { "name": "ForSideView", "type": "bool" },
+        { "name": "OnlyPlayerAttack", "type": "bool" },
+        { "name": "IsCrackup", "type": "bool" }
+      ]
+    },
+    "EnemySpinnerSpawner": {
+      "parent": "EnemySpinnerBaseSpawner",
+      "fields": [
+      ]
+    },
+    "ObjBigChaserEventCollisionSpawner": {
+      "parent": "EnemyEventCollisionBaseSpawner",
+      "fields": [
+        { "name": "Mode", "type": "ObjBigChaserEventCollisionSpawner::Mode" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "AttackType", "type": "ObjBigChaserEventCollisionSpawner::AttackType" }
+      ]
+    },
+    "EnemyBigChaserSpawner": {
+      "parent": "EnemyChaserBaseSpawner",
+      "fields": [
+        { "name": "InitialState", "type": "EnemyBigChaserSpawner::InitialState" },
+        { "name": "BombAtEndOfPath", "type": "bool" },
+        { "name": "AppearOffset1D", "type": "float32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "UpperEnable", "type": "bool" },
+        { "name": "JumpGuideEnable", "type": "bool" }
+      ]
+    },
+    "ObjEggChaserEventCollisionSpawner": {
+      "parent": "EnemyEventCollisionBaseSpawner",
+      "fields": [
+        { "name": "AutoRecruitSwitch", "type": "bool" }
+      ]
+    },
+    "EnemyChaserBaseSpawner": {
+      "parent": "EnemyBaseSpawner",
+      "fields": [
+        { "name": "Datum", "type": "EnemyChaserBaseSpawner::Datum" },
+        { "name": "Altitude", "type": "float32" }
+      ]
+    },
+    "EnemyEggChaserSpawner": {
+      "parent": "EnemyChaserBaseSpawner",
+      "fields": [
+        { "name": "TargetRange", "type": "float32" },
+        { "name": "RaycastRange", "type": "float32" },
+        { "name": "ExitCrashEnable", "type": "bool" },
+        { "name": "NumUnits", "type": "uint32" },
+        { "name": "NumMaxUnits", "type": "uint32" },
+        { "name": "AutoRecruitEnable", "type": "bool" },
+        { "name": "AutoLaneChangeEnable", "type": "bool" },
+        { "name": "TakedownEnable", "type": "bool" }
+      ]
+    },
+    "ObjMotoraEventCollisionSpawner": {
+      "parent": "EnemyEventCollisionBaseSpawner",
+      "fields": [
+        { "name": "AutoRecruitSwitch", "type": "bool" }
+      ]
+    },
```

Comment:
Unused Object. Parameter names based on in-game functionality and similar parameters specifically from `EggChaserEventCollision`.

More information on the object at:
https://hedgedocs.com/discoveries/sonic2010/objects/unused/#motoraeventcollision